### PR TITLE
Added __FILE_FULL_PATH__ to changelog.

### PR DESCRIPTION
**Pull request by marler8997**
https://issues.dlang.org/show_bug.cgi?id=16441

### CODE REVIEW

File: changelog.dd [hunk 1]
```diff
@@ -23,6 +23,7 @@ $(BUGSTITLE Language Changes,
 $(BUGSTITLE Compiler Changes,
     $(LI $(RELATIVE_LINK2 deferred_alias, Analysis for aliases in imported modules is deferred.))
     $(LI $(RELATIVE_LINK2 native_tls_osx, Native TLS on OS X 64 bit.))
+    $(LI $(RELATIVE_LINK2 __FILE_FULL_PATH__, Special trait replaced by the source file's absolute file name.)
```
Comment (JackStouffer) [inline changelog.dd]: `__FILE_FULL_PATH__ ` is not a trait, it's a manifest constant.

Comment (marler8997) [inline changelog.dd]: A manifest constant? I'm not familiar with the term, what's the definition?  I got "special trait" from https://dlang.org/spec/traits.html , but it looks like that page say's it's actually a "special keyword".

**Comment by mathias-lang-sociomantic** [comment]
Please also submit a dlang.org P.R.

**Comment by BBasile** [comment]
also update this https://github.com/dlang/dlang.org/blob/master/spec/lex.dd#L1066

File: changelog.dd (hunk 1 above)
Comment (marler8997) [inline changelog.dd]: Actually I just googled it.  It does look at behave like a manifest constant.  However, it isn't defined in the same way (must be defined by the compiler vs somewhere in the application's code base).  In the current compiler implementation, it required a new token be added, which makes it seem like a keyword.  What do you think?

Comment (BBasile) [inline changelog.dd]: It 's a keyword god damnit.

**Pull request merged**